docs/source/code-generation/codegen-cli.mdx

```diff
@@ -132,7 +138,7 @@ The Apollo iOS Codegen CLI is a command like utility for Apollo iOS code generat
 
 | Command | Description |
 | ------ | ----------- |
-| `init`         | Initialize a new configuration with defaults. |
+| `init`         | A prerequisite step to initialize a new configuration with defaults. |
```

Comment:
It's not really a prerequisite since you can use `--string` and enter the entire configuration string into the `generate` command. You could also have a config file elsewhere already. `init` is simply another command.

It might be one of the first steps in a Getting Started guide but not a prerequisite to code generation.

Reply:
With the getting started guide, I think that should address when `init` needs to be used. So I'll remove this.